openedx/core/djangoapps/programs/tests/mixins.py

```diff
@@ -125,6 +125,19 @@ class ProgramsDataMixin(object):
         }
     ]
 
+    PROGRAMS_CREDENTIALS_DISPLAY_DATA = [
```

Comment:
@ahsan-ul-haq almost. The goal is to avoid further use or extension of this mixin. Use your factory directly in your tests to create data only when it's necessary.